src/rgw/driver/rados/rgw_rados.cc

```diff
@@ -7953,15 +7953,15 @@ int RGWRados::get_bucket_info(RGWServices *svc,
 
 int RGWRados::try_refresh_bucket_info(RGWBucketInfo& info,
                                       ceph::real_time *pmtime,
-                                      const DoutPrefixProvider *dpp,
+                                      const DoutPrefixProvider *dpp, optional_yield y,
```

Comment:
@cbodley It's a knit, but there are dozens of instances in this PR where the `optional_yield` argument in a function definition is added on the same line as another argument even though every other argument in the function is on it's own line. I think these optional_yields should go on their own line. What do you think?